lms/djangoapps/sneakpeek_deeplink/tests.py

```diff
@@ -143,3 +155,18 @@ def test_sneakpeek_course_enrollment_closed(self):
         req.path = '/courses/closed/course/run/info'
         self.assertIsNone(self.middleware.process_request(req))
         self.assertNoSneakPeek(req, self.closed_course)
+
+    def test_deleted_course_with_preferences(self):
+        # initial sneakpeek for the created course should be successful
+        initial_req = self.make_successful_sneakpeek_login_request()
```

Comment:
This is an example where using unabbreviated variable names helps a bit more.

Is `req` "requirements" or "request"?
Is `res` "response" or "result"
Is `re` "request", "response", or "regex"?

While context is generally revealing, it's easier to read large blocks of code without relying on context.
And sometimes it's still not obvious.